tools/unicode_table_generator.d

```diff
@@ -921,6 +927,13 @@ void writeGraphemeTries(File sink)
     writeBest3Level(sink, "hangulLVT", hangul.table["LVT"]);
     writeBest3Level(sink, "mc", props["Mc"]);
     writeBest3Level(sink, "graphemeExtend", props["Grapheme_Extend"]);
+
+    // emoji related data
```

Comment:
Inaccurate comment. Prepend and control tables aren't (mostly) related to emojis, only the extended pictographic table is.

Reply:
Adjust as required, this PR exists to unblock you :)